consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/statemachine/QbftRound.java

```diff
@@ -127,6 +127,30 @@ public ConsensusRoundIdentifier getRoundIdentifier() {
     return roundState.getRoundIdentifier();
   }
 
+  /**
+   * Create a block
+   *
+   * @return a Block
+   */
+  public Block createBlock(final long headerTimeStampSeconds) {
```

Comment:
Could replace other places in `QbftRound.java` that currently call `blockCreator.createBlock()` with just `createBlock()` now that you've added this function. Similarly for `updateStateWithProposalAndTransmit()` - those can be replaced with `sendProposalMessage()` now?

Reply:
I don't know about that, I did wrote my changes following the "law of least effort", or the "if it ain't broke, don't fix it " rule... I remember that I split the `createAndSendProposalMessage `into to `createBlock `and `sendProposalMessage `but the original is still being invoked... Regarding changing calls I'm afraid \i don't know what the current code does and where they are invoked to do such change...

Reply:
Yeah just a suggestion to make the code more consistent and re-use your new methods. Not a huge issue, and it only relates to the other places in that class (`QbftRound.java`)

Reply:
I refactored some methods, some of them cannot be replaced, TBH I think as it was the code was more clear, the reader would see immediatly what each call was doinf instead of having to check one more method, but if it is a question of code style then it's fine by me.